### PR TITLE
Enable set syslog host for Filebeat HAProxy module

### PR DESCRIPTION
**Pull request by jasontedor**
The syslog input for the Filebeat HAProxy module binds by default to localhost:9001. It is possible to configure the port through the variable var.syslog_port, but it is not possible for the syslog_host. This can be useful when running Filebeat inside a Docker container. Since it is not easy to forward a port from the host to localhost inside the Docker container, it restricts the ability to use Filebeat in a Docker container to log HAProxy running on a remote instance. With this commit, we make it possible to configure the host that the syslog input for the HAProxy binds to through a variable var.syslog_host. This makes it possible to have HAProxy running on a remote instance (e.g., a FreeBSD 11.2 host), and configure that instance to remote log to a Filebeat instance running inside a Docker container.

### CODE REVIEW

**Comment by ruflin** [comment]
jenkins, test this

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -76,6 +76,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha1...master[Check the HEAD d
 *Filebeat*
 
 - Added `detect_null_bytes` selector to detect null bytes from a io.reader. {pull}9210[9210]
+- Added `syslog_host` variable to HAProxy module. {pull}9366[9366]
```
Comment (jsoriano) [inline CHANGELOG.asciidoc]: A small description of the purpouse of the new variable could be nice, though this can be obvious on this case.

Comment (jasontedor) [reply CHANGELOG.asciidoc]: I pushed a80f33f493a3fa87acdd8bf8d695f443d764cb6a.

Comment (jsoriano) [reply CHANGELOG.asciidoc]: Perfect, thanks!

**Comment by jasontedor** [comment]
@jsoriano I don't think the build failure had anything to do with my change. How do I trigger a new build?

**Comment by exekias** [comment]
jenkins, retest this please

**Pull request merged**